docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -424,3 +424,35 @@ If in secure mode, the secrets are retrieved from the secret store based on the
 
 If running in insecure mode, the secrets are retrieved from the *Writable.InsecureSecrets* configuration.
 
+### Background Publishing
+
+Application Services using the MessageBus trigger can request a background publisher using the AddBackgroundPublisher API in the SDK.  This method takes an int representing the background channel's capacity as the only parameter and returns a reference to a BackgroundPublisher.  This reference can then be used by background processes to publish to the configured MessageBus output.  Example usage is as follows:
+
+```go
+edgexSdk := &appsdk.AppFunctionsSDK{ServiceKey: serviceKey}
+if err := edgexSdk.Initialize(); err != nil {
+	edgexSdk.LoggingClient.Error(fmt.Sprintf("SDK initialization failed: %v", err))
+	os.Exit(-1)
+}
+
+//initialize background publisher with a channel capacity of 10
+pub := sdk.AddBackgroundPublisher(10)
+
+done := make(chan struct{})
+defer close(done)
+
+//pass the publisher to your jobs 
+for _, j := range jobs {
```

Comment:
I think it would be helpful to show example of Job using the publisher to send the data to complete the `picture`. Also `jobs` isn't define in sample so might want to change comment to  `pass the publisher to your jobs (defined elsewhere)` .

Reply:
I like this idea to show a more detailed example, I'll just code a simple loop to use it inline.

Reply:
I left the actual message construction as an exercise for the user, but added a simple method demonstrating usage of the publisher to the doc.  Tried not to make the code sample too much longer so I left out logging calls etc...